www/index.php

```diff
@@ -78,395 +78,396 @@
+                                                                $pIndex++;
+                                                            }
+                                                        }
+                                                        ?>
+                                                    </div>
+                                                </div>
+                                                <div class="test_presets_easy_checks">
+                                                    <div class="fieldrow" id="description"></div>
+                                                    <div class="fieldrow">
+                                                        <label for="rv"><input type="checkbox" name="rv" id="rv" class="checkbox" onclick="rvChanged()"> Include Repeat View <small>(Loads the page, closes the browser and then loads the page again)</small></label>
+                                                    </div>
+                                                    <div class="fieldrow">
+                                                        <label for="lighthouse-simple"><input type="checkbox" name="lighthouse" id="lighthouse-simple" class="checkbox"> Run Lighthouse Audit <small>(Runs on Chrome, emulated Moto G4 device, over simulated 3G Fast connection)</small></label>
+                                                    </div>
+                                                    <?php if ($is_paid) : ?>
+                                                        <div class="fieldrow">
+                                                            <label for="private-simple"><input type="checkbox" name="private" id="private-simple" class="checkbox"> Make Test Private <small>Private tests are only visible to your account</small></label>
+                                                        </div>
+                                                    <?php endif; ?>
+                                                </div>
+
+                                                <div class="test_presets_easy_submit">
+                                                    <?php if ($is_logged_in) : ?>
+                                                        <small class="test_runs <?= $remaining_runs === 0 ? 'runs-warn' : ''; ?>"><span><?= $remaining_runs; ?> Runs Left</span> | <a href="/account">Upgrade</a></small>
+                                                    <?php endif; ?>
+                                                    <input type="submit" name="submit" value="Start Test &#8594;" class="start_test" aria-disabled="<?= $remaining_runs == 0 ? true : false; ?>">
```

Comment:
I'd make this `remaining_runs <= 0`,  because distributed systems and eventual consistency. There's a chance we let people run past their allotted amount ¯\_(ツ)_/¯

Reply:
also, do we want to add a `disabled=disabled` attribute if the same condition is met?

Reply:
yes! I just noticed that aria-disabled is not cutting it when I QA'd it